README.md

```diff
@@ -2,6 +2,16 @@
 
 This Repository is a guide to all the git commands you need to use in case any of these situations arise
 
+## Index :books:
+* [Basic Commands :v:](#basic-commands-v)
+* [Branches :deciduous_tree:](#branches-deciduous_tree)
+* [Update forked Repository with original Repository :hourglass:](#update-forked-repository-with-original-repository--hourglass)
+* [Dev branch is X commits behind and Y commits ahead of master](#master)
+* [Short hands](#shorthands)
+* [GitHub CLI commands](#cli)
+
+
```

Comment:
It seems that these links are broken please fix them

Reply:
Okay sure! I'll add them back in.